streamer/input_configuration.py

```diff
@@ -243,25 +243,12 @@ def disallow_field(name: str, reason: str) -> None:
       disallow_field('start_time', reason)
       disallow_field('end_time', reason)
 
-    # A path to a pipe into which this input's contents are fed.
-    # None for most input types.
-    self._pipe: Optional[str] = None
 
-  def set_pipe(self, pipe: str) -> None:
-    """Set the path to a pipe into which this input's contents are fed.
-
-    If set, this is what TranscoderNode will read from instead of .name.
-    """
-
-    self._pipe = pipe
-
-  def get_path_for_transcode(self) -> str:
-    """Get the path which the transcoder will use to read the input.
-
-    For some input types, this is a named pipe.  For others, this is .name.
+  def reset_name(self, pipe: str) -> None:
```

Comment:
Nit: Now that we have a Pipe class, calling this param `pipe` makes it sound like it should be a Pipe object.  How about `path`?

Reply:
Oh, I overlooked this one.
since it's a string and not a pipe object, i agree with you it should be called `path` instead.
but for clarity - as this path is a path of a pipe and not a file on the file system - let's call it `pipe_path`.